tests/extension/oneapi_device_global/device_global_api_arrow_operator.cpp

```diff
@@ -130,9 +130,9 @@ class TEST_NAME : public sycl_cts::util::test_base {
    */
   void run(util::logger& log) override {
 #if !defined(SYCL_EXT_ONEAPI_PROPERTY_LIST)
-    log.skip("SYCL_EXT_ONEAPI_PROPERTY_LIST is not defined, test is skipped");
```

Comment:
Can we remove this function explicitly to prevent new usages in the future?

Reply:
It was removed during the Catch2 migration, the PR that introduced this likely just predates it!